scripts/common_click.inc

```diff
@@ -252,11 +252,12 @@ function drawWater()
     local water = srFindImage("iconWaterJugSmall.png", 1);
     if water then
       safeClick(water[0]+3, water[1]-5);
+	  lsSleep(click_delay*5);
       local max = waitForImage("crem-max.png", 500, "Waiting for Max button");

```

Comment:
That call to waitForImage has a 500ms timeout - I would agree that the timeout just needs to be increased if that's the goal - no need to put in a mandatory wait.